common/coat_of_arms/coat_of_arms/00_wc_landed_titles.txt

```diff
@@ -362,17 +362,6 @@ d_rustbolt = {
 	}
```

Comment:
Should be a bit higher.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/122671878-b9c90700-d1d9-11eb-82e8-489073acc9fb.png)

</details>